From f70fd29b8a5cb255a56927050a2448b5bb4b67f9 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 11 Aug 2021 22:22:25 +0200
Subject: [PATCH 32/35] Revert "block/mq-deadline: remove redundant assignment
 of variable ret"

This reverts commit cb29bee87de410543254c339b2875ef6e48cf4de.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 block/mq-deadline-main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/mq-deadline-main.c b/block/mq-deadline-main.c
index c250ff0fa..6f612e6dc 100644
--- a/block/mq-deadline-main.c
+++ b/block/mq-deadline-main.c
@@ -639,6 +639,7 @@ static int dd_init_sched(struct request_queue *q, struct elevator_type *e)
 	if (ret)
 		goto free_stats;
 
+	ret = 0;
 	q->elevator = eq;
 	return 0;
 
-- 
2.33.0.rc0.52.ge5a14ddd2d

